### ISSUE TITLE
VPN does not start during analysis

### ISSUE THREAD

**Issue by af104851b8** [kevoreilly/CAPEv2#362]
## This is opensource and you getting __free__ support so be friendly!

# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed
- [X] I'm reporting the issue to the correct repository (for multi-repository projects)

# Expected Behavior

The scheduler code should start the chosen VPN connection when a submitted sample is being analysed. 
The scheduler currently has code which automatically shuts down said VPN after the analysis.  

# Current Behavior

The scheduler code does not start the chosen VPN connection and seemingly has no code to do so. 
The scheduler shuts the VPN down if it has been started before manually. 

I've added the following to the the CAPEv2/lib/cuckoo/core/scheduler.py file:

```
if self.route in vpns:
    self.rooter_response = rooter("vpn_enable", self.route)
    time.sleep(1)
```

at line 535

After applying this code block, the behavior is as expected. 
This does seem like a dirty solution and I'm wondering what the correct way to handle VPNs is supposed to be. 

Further information about the modifications in my setup, related to VPNs:
/etc/iproute2/rt_tables added content:
```
100 tun0
101 tun1
```

/etc/openvpn/ added content:
provider1.conf
provider2.conf

CAPEv2/conf/routing.conf 
```
route = internet
internet = ens33
auto_rt = yes
enabled = yes
vpns= vpn0,vpn1
[vpn0]
name = provider1
description = provider1
interface = tun0
rt_table = tun0
[vpn1]
name = provider2
description = provider2
interface = tun1
rt_table = tun1

```
CAPEv2/conf/vpn.conf:
```
vpns= vpn0,vpn1
[vpn0]
name = provider1
description = provider1
interface = tun0
rt_table = tun0
[vpn1]
name = provider2
description = provider2
interface = tun1
rt_table = tun1
```

**Comment by doomedraven** [comment]
it not how that works, you should start that, as if 2 different vms want to use the same routing and 1 finish before the second than it kills the tunnel

**Comment by af104851b8** [comment]
I've figured it out, as per your response on https://github.com/kevoreilly/CAPEv2/issues/235. 
I overlooked the error message when the script was adding the default routes. 

For others who may have the same issue as I had:
The rt_table entry in routing.conf & vpn.conf and /etc/iproute2/rt_tables has to be the common name of your vpn server and not some tunnel interface you'd like to use. 
Then you just run: 
sudo openvpn --config /etc/openvpn/provider1.conf --script-security 2 --route-noexec --route-up utils/route.py &
sudo openvpn --config /etc/openvpn/provider2.conf --script-security 2 --route-noexec --route-up utils/route.py &
